recipe/meta.yaml

```diff
@@ -96,4 +104,4 @@ extra:
     - jjhelmus
     - msarahan
     - pelson
-    - ocefpaf
```

Comment:
Guessing this was accidental.

Reply:
yeah guess AnacondaRecipes version missed this one.